src/main/java/com/faforever/client/notification/TransientNotificationController.java

```diff
@@ -107,6 +109,10 @@ public Region getRoot() {
   }
 
   public void onClicked(MouseEvent event) {
-    action.call(event);
+    if (event.getButton().equals(MouseButton.SECONDARY)) {
+      dismiss();
+    } else {
+      action.call(event);
+    }
```

Comment:
Can we have a unit test for this, which checks that the correct thing happens based on which mouse button has been clicked?